bundles/dna_templates/agent/dna.yaml

```diff
@@ -26,11 +26,13 @@ integrity:
         - extern_id: index_agent_inventoried_economic_resources
           allowed_method: [agent_index, index_agent_inventoried_economic_resources]
   zomes:
+    # application zomes
+    - name: agent_integrity
+      bundled: "<repository-path>/target/wasm32-unknown-unknown/release/hc_zome_rea_agent_integrity.wasm"
+
     # utility zomes
     - name: remote_auth_integrity
       bundled: "<repository-path>/target/wasm32-unknown-unknown/release/hc_zome_dna_auth_resolver_hrea_integrity.wasm"
-    - name: agent_integrity
-      bundled: "<repository-path>/target/wasm32-unknown-unknown/release/hc_zome_rea_agent_integrity.wasm"
 
 coordinator:
   zomes:
```

Comment:
You're going to need to add something like this, similar to how it is done for `remote_auth`
```
      dependencies:
        - name: agent_integrity
```